Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1116,7 +1102,6 @@ + (void)startWithUserId:(NSString *)userId
     } repeats:YES];
 
     // Extension close.
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 80000
```

Comment:
Was our code not working at all for below 8?

Reply:
i guess so?

Reply:
dang